src/pages/lib/graphqlapi/mutate-data/q/platform/[platform].mdx

```diff
@@ -3,7 +3,7 @@ export const meta = {
   description: `Learn more about how to create and update data using GraphQL APIs in Amplify`
 };
 
-In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client.
+In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client. 
 
 Before you begin, you will need:
 
```

Comment:
If Connect guide is not ready - the JS link below will need to be updated to getting started page.